### PR TITLE
use quarkus-micro instead of quarkus-distroless

### PR DESCRIPTION
**Pull request by matthyx**
**Check list**:

Your pull request:

- [x] targets the `development` branch
- [ ] uses the `999-SNAPSHOT` version of Quarkus
- [ ] has tests (`mvn clean test`)
- [ ] works in native (`mvn clean package -Pnative`)
- [ ] has native tests (`mvn clean verify -Pnative`)
- [ ] makes sure the associated guide must not be updated
- [ ] links the guide update pull request (if needed)
- [ ] updates or creates the `README.md` file (with build and run instructions)
- [ ] for new quickstart, is located in the directory _component-quickstart_
- [ ] for new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

**Comment by Karm** [comment]
@matthyx Lemme try the AWT one to see if it survived the migration...

**Comment by matthyx** [comment]
> @matthyx Lemme try the AWT one to see if it survived the migration...

any update?

**Review by Karm — CHANGES_REQUESTED** [review]
@matthyx It doesn't work for me :-(

When I go through the motions in `awt-graphics-rest-quickstart/src/main/docker/Dockerfile.native-distroless`

It ends up with:

```
docker: Error response from daemon: unable to find user nonroot: no matching entries in passwd file.
```

**Comment by matthyx** [comment]
Damn, how can I test it?

**Comment by Karm** [comment]
@matthyx Well...it's all in the file: awt-graphics-rest-quickstart/src/main/docker/Dockerfile.native-distroless

Is it isolated to this particular quickstart?  I noticed your change:

```
- USER 1001
+ USER nonroot
```

and that expects the new runtime container you are swapping all those containers for to actually have such user created...

**Comment by matthyx** [comment]
> @matthyx Well...it's all in the file: awt-graphics-rest-quickstart/src/main/docker/Dockerfile.native-distroless
> 
> Is it isolated to this particular quickstart? I noticed your change:
> 
> ```
> - USER 1001
> + USER nonroot
> ```
> 
> and that expects the new runtime container you are swapping all those containers for to actually have such user created...

my mistake, actually should be `1001` everywhere... I will correct later, sorry!

**Comment by matthyx** [comment]
@Karm fixed, thanks for the review!

File: awt-graphics-rest-quickstart/src/main/docker/Dockerfile.native-distroless [hunk 1]
```diff
@@ -1,24 +1,4 @@
...
-#
-# Using Debian based https://github.com/quarkusio/quarkus-images/blob/main/quarkus-distroless-image.yaml#L16
-# will not do for this Quickstart as only libz.so is copied and the runtime
-# might not be compatible with Quarkus executable built using UBI image.
-#
-# The undermentioned example uses ubi-micro image and manually copies necessary
-# libraries and files from ubi-minimal image.
-#
-# See https://www.redhat.com/en/blog/introduction-ubi-micro on how to start using Buildah.
-#
```
Comment (Karm) [inline awt-graphics-rest-quickstart/src/main/docker/Dockerfile.native-distroless:18]: @matthyx May I ask you to leave the info there, just deleting 
```
# Using Debian based https://github.com/quarkusio/quarkus-images/blob/main/quarkus-distroless-image.yaml#L16
# will not do for this Quickstart as only libz.so is copied and the runtime
# might not be compatible with Quarkus executable built using UBI image.
```

The pointer about buildah is still relevant. Copying files manually in this fashion does the job for a simple example, for the sake of brevity, but it won't do for any serious long term maintainable project.

**Review by Karm — APPROVED** [review]
Thanks! It works for me as far as AWT quickstart is concerned. 

One note about comment in the Dockerfile. 

Cheers

**Comment by matthyx** [comment]
> Thanks! It works for me as far as AWT quickstart is concerned.
> 
> One note about comment in the Dockerfile.
> 
> Cheers

I have restored your ASCII art ;-)
Thanks for the review!

@cescoffier PTAL

**Review by cescoffier — COMMENTED** [review]
I would have changed the file names to micro. Except that looks good to me.

**Comment by matthyx** [comment]
> I would have changed the file names to micro. Except that looks good to me.

done, sorry for the delay - I hope it's not too late for 2.6

**Review by cescoffier — APPROVED** [review]

**Pull request merged**